torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3250,7 +3248,17 @@ def _gather(shape, index_dim, max_indices):
         (_tensor(()), (0, zero.clone().detach(), 2.5)),
     )
 
-    return [SampleInput(tensor, args=args) for tensor, args in test_cases]
+    samples = []
+    for tensor, args in test_cases:
+        samples.append(SampleInput(tensor, args=args, kwargs={'reduce': None}))
```

Comment:
Neat; would you `.clone().detach()` `tensor` for the two extra sample inputs so each sample input has an independent `input` tensor?